company/getAll.js

```diff
@@ -0,0 +1,26 @@
+import { failure, resourceNotFound, success } from '../libs/response-lib';
+import * as dynamoDbLib from '../libs/dynamodb-lib';
+
+export const main = async () => {
+	const params = { TableName: process.env.COMPANIES_TABLE_NAME };
+
+	let companies = [];
+	let items;
```

Comment:
Seems like `companies` and `items` could be declared as `const` inside the `try` block